manifests/charts/istio-operator/templates/deployment.yaml

```diff
@@ -30,6 +30,7 @@ spec:
           command:
           - operator
           - server
+          - --max-concurrent-reconciles={{ .Values.operator.maxConcurrentReconciles }}
```

Comment:
Why do we need to expose this? Are you applying many IOPs simultaneously?

Reply:
yes

Reply:
I don't mind adding this, but my only concern is: when referring to https://github.com/istio/enhancements/issues/166, it appears that the operator controller is heading towards deprecation. I'm concerned this might confuse users, especially since new features are still being added to something that is no longer encouraged.

Reply:
that's a good point